src/current/_includes/releases/v24.2/v24.2.5.md

```diff
@@ -0,0 +1,118 @@
+
+<h3 id="v24-2-5-general-changes">General changes</h3>
+
+- Change the license cockroach is distributed under to the new CockroachDB Software License. [#131707][#131707] [#131956][#131956] [#131961][#131961] [#131983][#131983] [#132011][#132011] [#132013][#132013] [#132012][#132012] [#132015][#132015] [#132014][#132014] [#132016][#132016] [#132054][#132054] [#131970][#131970] [#132056][#132056] [#132801][#132801] [#132704][#132704]
+- The cluster setting `diagnostics.reporting.enabled` is now ignored if the cluster has an Enterprise Trial or Enterprise Free license or if the license cannot be loaded. [#132464][#132464]
+- The new metrics `changefeed.sink_errors` and `changefeed.internal_retry_message_count` allow you to observe the rate of and internal retries for a sink. [#132353][#132353]
+- Added a timer for inner sink client flushes. [#133197][#133197]
+
+<h3 id="v24-2-5-db-console-changes">DB Console changes</h3>
+
+- The DB Console now shows a warning if the cluster is throttled or will be throttled soon due to an expired Enterprise Free or Enterprise Trial license or missing telemetry data. Clusters with an Enterprise license are not throttled. [#132093][#132093]
+- The Range Count column on the Databases page is no longer shown due to performance issues. This data is still available via the `SHOW RANGES` command. [#133268][#133268]
+
+<h3 id="v24-2-5-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug where timers were not correctly registered with the metric system. [#133197][#133197]
+- Fixed a bug where the command-line interface would not correctly escape JSON values that had double quotes inside a string when using the `--format=sql` flag. [#131931][#131931]
+- Fixed an error that could occur if a `SET` command used an aggregate function as the value. [#131960][#131960]
+- Fixed a bug where ordering by `VECTOR` columns could result in an internal error in some cases. Now an `unimplemented` error is returned instead. [#132107][#132107]
+- Added automated clean-up/validation for dropped roles inside default privileges. [#132136][#132136]
+- Fixed a bug that that caused incorrect evaluation of a `CASE`, `COALESCE`, or `IF` expression with a branch that produced fixed-width string-like types, such as `CHAR`. [#130889][#130889]
+- Fixed a bug that could cause the `BPCHAR` type to incorrectly impose a length limit of 1. [#130889][#130889]
+- Fixed a rare bug that could prevent a backup from being restored and could cause the error `rewriting descriptor ids: missing rewrite for <id> in SequenceOwner...`. This bug could occur only if a `DROP COLUMN` operation dropped a sequence while the backup was running. [#132326][#132326]
+- Fixed a bug introduced in v23.1 that could cause incorrect results if a join evaluates columns with equivalent but not identical types, such as `OID` and `REGCLASS`, for equality. The join must lookup into an index that includes a computed column that references one of the equivalent columns. [#132508][#132508]
```

Comment:
```suggestion
```suggestion
- Fixed a bug introduced in v23.1 that could cause incorrect results if a join evaluates columns with equivalent but non-identical types, such as `OID` and `REGCLASS`, for equality. The issue arises when the join performs an index lookup on an index that includes a computed column referencing one of the equivalent columns. [#132508][#132508]
```
```